ci-operator/step-registry/cucushift/ota/postupgrade/cucushift-ota-postupgrade-commands.sh

```diff
@@ -159,6 +159,23 @@ function post-OCP-47197(){
     return 0
 }
 
+function post-OCP-56083(){
+    echo "Test Start: OCP-56083"
+    echo "Upgrade cluster when channel is unset"
+    local expected_msg expected_msg
+    tmp_log=$(mktemp)
+    expected_msg="Configured channel is unset"
+    oc get clusterversion -ojson | jq '.items[].status.conditions[]|select(.type == \"ReleaseAccepted\")' 2>&1 | tee "${tmp_log}" 
```

Comment:
Sounds like this check is equivalent to https://gitlab.cee.redhat.com/aosqe/v4-installer-autotest/-/blob/master/cases/TestCases/OCP-56083/upgrade_nochannel_risk.py#L53 , shall we use a different expected message for it?